<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowHistoricData</class>
 <widget class="QDialog" name="ShowHistoricData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Historic Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutConstituent">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Variable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Start Time/Date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="displayFormat">
          <string>M/d/yyyy h:mm AP</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>End Time/Date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit_2">
         <property name="displayFormat">
          <string>M/d/yyyy h:mm AP</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Refresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalGraph"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
